test/test_helper.rb

```diff
@@ -1,3 +1,5 @@
+require "simplecov"
+SimpleCov.start
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
```

Comment:
We actually have great coverage, I think we should talk about how we can get it to 100 percent tomorrow.